src/_component/about/about.scss

```diff
@@ -0,0 +1,120 @@
+.section__ {
+
+  &header-wrapper{
+    padding: 10px;
+    height: 60px;
+    background-color: #d52027;
+    box-shadow: 0 3px 6px rgba($color: #000000, $alpha: 0.30);
+    position: relative;
+    
+    &::before{
+      position: absolute;
+      content: '';
+      background-color: #d52027;
+      left: -36px;
+      top: 0;
+      bottom: 0;
+      right: -36px;
+    }
+    
+    &::after{
+      position: absolute;
+      content: '';
+      box-shadow: 0 3px 6px rgba($color: #000000, $alpha: 0.30);
+      left: 0;
+      top: 0;
+      bottom: 0;
+      right: 0;
+    }
+  }
+  
+  &header{
+    height: 40px;
+    line-height: 40px;
+    text-align: center;
+    color: white ;
+    position: relative;
+    font-size: 24px;
+  }
+
+  &main-title,
+  &sub-title {
+    color: #707070;
+    text-align: center;
+  }
+
+  &main-title{
```

Comment:
Just for readability, you may need an empty space between the class name and `{`